### PR TITLE
Lesson5

### PR DESCRIPTION
**Pull request by saumya665**
files: xmlpublic.html and xml.js in lesson5

### CODE REVIEW

File: xmlpublic.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<html lang = "en">
+  <head>
+    <meta charset = "UTF-8">
+    <title></title>
+    </head>
+    <body>
+      Name: <input type = 'text' id = 'a1'/>
```
Comment (pkdotson) [inline xmlpublic.html]: I would name the id's something more specific.

Comment (evo777) [reply xmlpublic.html]: I would name the id to like `name` or `comment`.

**Review by evo777 — APPROVED** [review]

File: lesson5/public/index.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<html lang = "en">
+  <head>
+    <meta charset = "UTF-8">
+    <title></title>
+  </head>
+  <body>
+    Name: <input type = 'text' id = 'a1'/>
```
Comment (chocobrownie) [inline lesson5/public/index.html]: a1 & a2 should be more readable variables

File: lesson5/xml.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+const express = require("express");
```
Comment (chocobrownie) [inline lesson5/xml.js:2]: you only need one xml.js

File: xml.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const fs = require("fs");
+const app = express();
+
+app.listen(3615);
+app.use(express.static('public'));
+
+app.get("/XMLHttpRequest",(req,res)=>{
+  console.log(req.query);
```
Comment (chocobrownie) [inline xml.js]: console.log is for debuging, it shouldn't appear in your result

File: lesson5/public/index.html [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    Comment: <input type = 'text' id = 'a2'/>
+    <input type = 'submit' id = 'b1'>
+    <script>
```
Comment (chocobrownie) [inline lesson5/public/index.html]: weird indentation

**Review by MadhuBeesetty — APPROVED** [review]
awesome

File: lesson5/public/index.html [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+    Name: <input type = 'text' id = 'nameBox'/>
+    Comment: <input type = 'text' id = 'commentBox'/>
+    <input type = 'submit' id = 'submitBox'>
+    <script>
+const button = document.getElementById('submitBox');
+button.onclick =()=>{
+  const name = document.getElementById('nameBox').value;
+  const comment = document.getElementById('commentBox').value;
+  const xhttp = new XMLHttpRequest();
+  xhttp.open('GET',`/XMLHttpRequest?nameBox=${name}&commentBox=${comment}`);
```
Comment (chocobrownie) [inline lesson5/public/index.html]: nameBox and commentBox better changed to nameBoxValue or something. They don't have to match the id name, actually it's better that they don't

File: lesson5/public/index.html [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+  xhttp.send();
+}
```
Comment (chocobrownie) [inline lesson5/public/index.html]: weird indentation

File: lesson5/xml.js [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+const fs = require("fs");
+const app = express();
+
+app.listen(3615);
+app.use(express.static('public'));
+app.get("/XMLHttpRequest",(req,res)=>{
+  fs.appendFile('/home/sya/garageScript/lesson5/public/xmlPublic.txt', `Name: ${req.query.nameBox} Comment: ${req.query.commentBox}`);
+    res.send('done');
```
Comment (chocobrownie) [inline lesson5/xml.js]: weird indentation

Comment (saumya665) [reply lesson5/xml.js]: fixed indentation with command gg=G

File: lesson5/xml.js (hunk 3 above)
Comment (saumya665) [reply lesson5/xml.js:2]: deleted the other xml.js files

File: lesson5/public/index.html (hunk 7 above)
Comment (saumya665) [reply lesson5/public/index.html]: fixed indentation

**Review by chocobrownie — APPROVED** [review]
Looks good to me

**Pull request merged**